ARM-wvd-templates/HCI/AddHciVirtualMachinesTemplate.json

```diff
@@ -90,20 +90,20 @@
       "customLocationId": {
         "type": "string",
         "metadata": {
-          "description": "A deployment target created and customized by your organization for creating virtual machines. The custom location is associated to an Azure Stack HCI cluster."
+          "description": "A deployment target created and customized by your organization for creating virtual machines. The custom location is associated to an Azure Stack HCI cluster. E.g., /subscriptions/<subscriptionID>/resourcegroups/Contoso-rg/providers/microsoft.extendedlocation/customlocations/Contoso-CL."
         }
       },
       "virtualProcessorCount": {
         "type": "int",
         "metadata": {
-          "description": "Virtual Processor Count"
+          "description": "Virtual Processor Count. Default is 4vCPU."
```

Comment:
maybe make this clear what they customer should use.   e,g, "4" is the default.  not "4vCPU".

Reply:
Makes sense, let's change it.